src/elements/alert/alert/alert.ts

```diff
@@ -173,7 +177,7 @@ class SbbAlertElement extends SbbIconNameMixin(SbbOpenCloseBaseElement) {
                   ></sbb-divider>
                   <sbb-transparent-button
                     negative
-                    size=${this.size === 'l' ? 'm' : this.size}
+                    size=${isLean() ? 's' : this.size === 'l' ? 'm' : this.size}
```

Comment:
Maybe I don't see anything, but of size is set to s (lean), the fallback of "this.size" is already kicking in (which correctly sets it)
```suggestion
                    size=${this.size === 'l' ? 'm' : this.size}
```